src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-description {
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+
+    @include transition;
+  }
+
+  &__product-description:hover {
+    color: $title-active-color;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      color: $secondary-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $main-color;
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: block;
+    height: 40px;
+    width: 100%;
+    line-height: 16px;
+    background-color: #00acdc;
```

Comment:
Consider using a variable for the background color (`#00acdc`) to maintain consistency and ease future changes. This will also improve maintainability if the color is used elsewhere.